functests/performance/rt-kernel.go

```diff
@@ -59,4 +60,53 @@ var _ = Describe("[performance]RT Kernel", func() {
 
 	})
 
+	It("[test_id:28526][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Non worker-cnf node should not have RT kernel installed", func() {
+		var newWorkerNode *corev1.Node
+		newRole := "worker-test"
+		newLabel := fmt.Sprintf("%s/%s", testutils.LabelRole, newRole)
+
+		By("Skipping test if cluster does not have another available worker node")
+		workerNodes, err := nodes.GetByRole(testclient.Client, "worker")
+		Expect(err).ToNot(HaveOccurred())
+
+		for _, node := range workerNodes {
+			if _, ok := node.Labels[fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerRT)]; ok {
+				continue
+			}
+			newWorkerNode = &node
+			break
+		}
+		if newWorkerNode == nil {
+			Skip("Skipping test because there are no additional worker nodes")
+		}
+		newWorkerNode.Labels[newLabel] = ""
+		Expect(testclient.Client.Update(context.TODO(), newWorkerNode)).ToNot(HaveOccurred())
+
+		Eventually(func() string {
```

Comment:
In my opinion we don't  need to run pod in this test. I hope it's enough to verify kernel directly on the node, in this case this "eventually" block can be changed to:

			cmd = []string{"uname", "-a"}
			Expect(execCommandOnWorker(cmd, &node)).NotTo(ContainSubstring("PREEMPT RT"), "Node should have non-RT kernel")

Reply:
In that case, we will need to change the previous test case as well.